File: 2310817210014_Damarjati Suryo Laksono/Modul 6/C/PRAK601-2310817210014-DamarjatiSuryoLaksono.c
#include <stdio.h>

int main() {
    int i, j, baris, kolom;
    scanf("%d %d", &baris,&kolom);
    int matriks[baris][kolom];
    for(i = 0; i < baris; i++) {
        for(j = 0; j < kolom; j++) {
            scanf("%d", &matriks[i][j]);
        }
    }
    for(i = 0; i < baris; i++) {
        for(j = 0; j < kolom; j++) {
            printf("%d ", matriks[i][j]);
        }
        printf("\n");
    }
    return 0;
}

File: Pemrograman-1/Modul 6/Python/PRAK601-2310817210014-DamarjatiSuryoLaksono.py
baris, kolom = map(int, input().split())
input_matriks = list(map(int, input().split()))
matriks = [[0] * kolom for _ in range(baris)]
index = 0
for i in range(baris):
    for j in range(kolom):
        matriks[i][j] = input_matriks[index]
        index += 1
for i in range(baris):
    for j in range(kolom):
        print(matriks[i][j], end=" ")
    print()
